### PR TITLE
Improve extensibility for MREF default template

### PR DESCRIPTION
**Pull request by vicancy**

### CODE REVIEW

File: src/docfx.website.themes/default/ManagedReference.html.primary.tmpl [hunk 1]
```diff
@@ -48,6 +48,9 @@
             {{#isEnum}}
               {{>partials/enum}}
             {{/isEnum}}
+            {{#_customContent}}
```
Comment (superyyrrzz) [inline src/docfx.website.themes/default/ManagedReference.html.primary.tmpl]: >_customContent [](start = 15, length = 14)

_isCustomContent?

File: src/docfx.website.themes/default/ManagedReference.html.primary.tmpl [hunk 2]
```diff
@@ -48,6 +48,9 @@
...
+              {{>partials/customContent}}
```
Comment (superyyrrzz) [inline src/docfx.website.themes/default/ManagedReference.html.primary.tmpl]: >customContent [](start = 26, length = 13)

add an empty file?

Comment (vicancy) [inline src/docfx.website.themes/default/ManagedReference.html.primary.tmpl]: by default _customContent is false, so I don't think empty file is needed

Comment (vicancy) [inline src/docfx.website.themes/default/ManagedReference.html.primary.tmpl]: Or add an empty file instead of using `_customContent`

**Comment by hellosnow** [comment]
:shipit:

**Pull request merged**